src/components/table-hoc/tests/TableRowNumberColumn.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import {shallow} from 'enzyme';
+import * as React from 'react';
+import {TableRowNumberColumn} from '../TableRowNumberColumn';
+
+describe('TableRowNumberColumn', () => {
+    it('should render without error', () => {
+        expect(() => shallow(<TableRowNumberColumn number={1} />)).not.toThrow();
+        expect(() => shallow(<TableRowNumberColumn number={'10000'} />)).not.toThrow();

```

Comment:
no need for the `{}` since you're using a string (but I think a children would be more flexible)

Reply:
it's now a react node, but I added the children prop